src/storage/admin/RebuildEdgeIndexProcessor.cpp

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "kvstore/LogEncoder.h"
+#include "storage/StorageFlags.h"
+#include "storage/admin/RebuildEdgeIndexProcessor.h"
+
+namespace nebula {
+namespace storage {
+
+void RebuildEdgeIndexProcessor::process(const cpp2::RebuildIndexRequest& req) {
```

Comment:
This file have replaced by RebuildEdgeIndexTask?

Reply:
Good catch, add by mistake, fixed